linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -238,28 +238,56 @@ if (env.performance == "yes") {
 
 if (env.sendMail == "yes"){
     node ("meta_slave") {
+        echo "Send email with kernel validation results link"
+        if (env.kernel == "linux-azure") {
+            version_identifier = "_azure"
+        } else {
+            version_identifier = "_edge"
+        }
+        kernels_info = readFile (LATEST_VERSION_LOCATION)
+        kernels_info = kernels_info.split("\n")
```

Comment:
make sure that the kernels_info is not null or empty, to not send useless information.

Reply:
will change